newsfragments/3317.feature

```diff
@@ -0,0 +1 @@
+Verified fakes for (some of) Tahoe's WebUI via treq
```

Comment:
Perhaps "allmydata.testing.web, a new module, now offers a supported Python API for testing Tahoe-LAFS web API clients."?

Also, do we need to say something about there being a supported Python API in Tahoe-LAFS somewhere now?

Also, do we want some prose/howto/tutorial-style docs for testing a Tahoe-LAFS web API client now?

Reply:
...but there isn't a supported Python API?

(I at least filed https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3318#ticket for docs .. maybe it's worth "trying it out" in magic-folder some more and use that as the example, once it's in master there?)